infra.bs

```diff
@@ -1522,6 +1551,82 @@ given a JavaScript value <var>jsValue</var>:
  <li><p>Return |result|.
 </ol>
 
+<p>To <dfn export lt="serialize an Infra value to a JSON string|serializing an Infra value to a JSON string">serialize an Infra value to a JSON string</dfn>,
+given a <a>string</a>, <a>boolean</a>, number, null, <a>list</a>, or <a>string</a>-keyed <a>map</a>
+|value|:
+
+<ol>
+ <li><p>Let |jsValue| be the result of
+ <a>converting an Infra value to a JSON-compatible JavaScript value</a>, given |value|.
+
+ <li>
+  <p>Return ! [$Call$](<a>%JSON.stringify%</a>, undefined, « |jsValue| »).
+
+  <p class=note>Since no additional arguments are passed to <a>%JSON.stringify%</a>, the resulting
+  string will have no whitespace inserted.
+</ol>
+
+<p>To <dfn export lt="serialize an Infra value to JSON bytes|serializing an Infra value to JSON bytes">serialize an Infra value to JSON bytes</dfn>,
+given a <a>string</a>, <a>boolean</a>, number, null, <a>list</a>, or <a>string</a>-keyed <a>map</a>
+|value|:
+
+<ol>
+ <li><p>Let |string| be the result of <a>serializing an Infra value to JSON bytes</a>, given
+ |value|.
+
+ <li><p>Return the result of running <a>UTF-8 encode</a> on |string|. [[!ENCODING]]
+</ol>
+
+<p>To <dfn lt="convert an Infra value to a JSON-compatible JavaScript value|converting an Infra value to a JSON-compatible JavaScript value">convert an Infra value to a JSON-compatible JavaScript value</dfn>,
+given |value|:
+
+<ol>
+ <li><p>If |value| is a <a>string</a>, <a>boolean</a>, number, or null, return |value|.
```

Comment:
The remaining primitives are bytes and code points. Bytes, although defined in Infra as "a sequence of eight bits" (\**shudders*\*), are basically used everywhere as numbers, so they should be serialized as such. But should code points be serialized as a string?